hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/AbstractRootedOzoneFileSystemTest.java

```diff
@@ -299,6 +299,11 @@ void testOzoneFsServiceLoader() throws IOException {
         OzoneConsts.OZONE_OFS_URI_SCHEME, confTestLoader), RootedOzoneFileSystem.class);
   }
 
+  @Test
+  void testUserHomeDirectory() {
+    assertEquals(new Path("/user/" + USER1), userOfs.getHomeDirectory());
```

Comment:
And similarly:

```
expected: </user/regularuser1> but was: <ofs://localhost:15004/user/regularuser1>
```

https://github.com/SaketaChalamchala/ozone/actions/runs/9212326470/job/25344354470#step:5:2184